src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -413,7 +413,7 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
                 }}
               >
                 <ThemedText.SubHeader color="white" lineHeight="20px">
-                  <span>{assetInBag ? 'Remove' : 'Add to Bag'}</span>
+                  <span data-testid="nft-details-add-to-bag">{assetInBag ? 'Remove' : 'Add to Bag'}</span>
```

Comment:
nit: name doesn't reflect that it can be remove too...maybe "nft-details-add-or-remove-from-bag"